keccak256/src/permutation/rho.rs

```diff
@@ -146,7 +127,11 @@ mod tests {
 
             fn configure(meta: &mut ConstraintSystem<F>) -> Self::Config {
                 let state: [Column<Advice>; 25] = (0..25)
-                    .map(|_| meta.advice_column())
+                    .map(|_| {
+                        let col = meta.advice_column();
+                        meta.enable_equality(col);
```

Comment:
If we enable the copy in the `configure` we don't have to do it here. Doing it here requires the consumer of the config to remember to do it.

Reply:
Hmmm I see your point.

Makes sense! Changing it!